library/src/com/google/maps/android/kml/KmlBoolean.java

```diff
@@ -0,0 +1,13 @@
+package com.google.maps.android.kml;
```

Comment:
Can you please add a copyright header to this file? Copy it from an existing file and bump the year to 2016.